packages/pxweb2-ui/src/lib/components/Tag/Tag.module.scss

```diff
@@ -8,20 +8,35 @@
 }
 
 .medium {
-  padding: 4px 8px;
+  height: 32px;
+  padding: 4.5px 8px 4px 8px;
   border-radius: var(--px-border-radius-small);
 }
 
 .small {
-  padding: 1px 6px 2px 6px;
+  height: 24px;
+  padding: 1.5px 4px 3px 4px;
```

Comment:
Is it possible to not use subpixels? If I recall correctly browsers render subpixels differently.

If there is no way around this, then leave it as is.

Reply:
Discussed with designer, we agreed to round up to 2 pixels.